jdbc/src/main/java/io/kestra/jdbc/runner/JdbcExecutor.java

```diff
@@ -298,7 +298,7 @@ private void executionQueue(Either<Execution, DeserializationException> either)
             Executor executor = new Executor(execution, null).withFlow(flow);
 
             // queue execution if needed (limit concurrency)
-            if (execution.getState().isCreated() && flow.getConcurrency() != null) {
+            if (execution.getState().getCurrent() == State.Type.CREATED && flow.getConcurrency() != null) {
```

Comment:
Isnt' it the same?

Reply:
No `QUEUED` is not contains on the `isCreated` status

Reply:
You replace `isCreated` by `getCurrent() == CREATED` so you didn't check the `QUEUED` status

Reply:
OK, just seen the same issue on EE, isCreated contains also the RESTARTED state which causes issue for the Pause task.